### DISCUSSION TITLE
Plugins for external display

### DISCUSSION THREAD

**Discussion by Rob-FolkDance** [quodlibet/quodlibet#4199] [Q&A] [answered]
Is there a plugin to show an external playlist with the dance playing at the top of the list and it keeps updating? The HTML plugin is a static list of the playlist.

**Comment by declension** [comment, marked as answer]
No, but you can [create new plugins quite easily](https://quodlibet.readthedocs.io/en/latest/development/plugins.html) (if you know some Python).
